go.mod

```diff
@@ -13,3 +13,11 @@ require (
 	github.com/spf13/pflag v1.0.5
 	github.com/stretchr/testify v1.8.0
 )
+
+require (
```

Comment:
I don't think this indirect requires lines are required. would you please remove them all and rerun `go mod tidy` so it will update the files? (It will update the version of `plush` which was released recently)